### PR TITLE
Governance docs per CE PR 1226

### PR DESCRIPTION
**Pull request by duglin**
See: https://github.com/cloudevents/spec/pull/1226

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1 @@
+TBD
```
Comment (duglin) [inline RELEASING.md]: ping @jlaswell for some help on what to put here

**Comment by duglin** [comment]
ping @jlaswell

**Comment by GrahamCampbell** [comment]
Releasing here is quite easy. Just cut a release on the GitHub UI, making sure to prefix the tag name with `v`.

**Comment by GrahamCampbell** [comment]
The only other relevant thing to releases is that if a minor or major change is merged into main, we need to change the branch alias in https://github.com/cloudevents/sdk-php/blob/main/composer.json#L77.

**Comment by duglin** [comment]
@GrahamCampbell Can you suggest some text and I'll add it?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+```console
+git fetch upstream
+git reset --hard upstream/main
+git checkout FETCH_HEAD
+git checkout -b 48-fix-http-agent-error
+```
+
+## Commit Messages
+
+Please follow the
```
Comment (jlaswell) [inline CONTRIBUTING.md]:
```suggestion
We prefer that contributors follow the
```

File: MAINTAINERS.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Maintainers
+
+Current active maintainers of this SDK:
+
+- [John Laswell](https://github.com/jlaswell)
```
Comment (jlaswell) [inline MAINTAINERS.md:5]: What's the process to get @GrahamCampbell onto this list as well, @duglin ?

Comment (duglin) [reply MAINTAINERS.md:5]: Per https://github.com/cloudevents/spec/blob/main/docs/SDK-GOVERNANCE.md#new-maintainers you can start a vote to add him.

Comment (jlaswell) [reply MAINTAINERS.md:5]: Thank you. Sorry, I should have spotted this.

**Pull request merged**